src/components/Icon/Icon.vue

```diff
@@ -0,0 +1,80 @@
+<template lang="pug">
+span(:class="className")
+  Text(
+    v-if="accessibilityLabel",
+    as="span",
+  )
```

Comment:
Is this missing content of Text here :thinking: ?

Reply:
Also missing `variant` as required property

Reply:
Text component - New Version: variant props is not required